### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by mtrezza**
## Release 2.2.0

The major work was to update the changelog for PRs going back more than 1 year.
Going forward, we have to make sure that a changelog entry is mandatory for every PR.
The PR template already includes a TODO for this, but there are old PRs not yet using the template.

### CODE REVIEW

**Review by fn-faisal — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+## Fixes
+
+# 2.2.0
+[Full Changelog](https://github.com/parse-community/parse-dashboard/compare/2.1.0...2.2.0)
+
+## New Features
+- Added data export in CSV format for classes (Cory Imdieke, Manuel Trezza) [#1494](https://github.com/parse-community/parse-dashboard/pull/1494)
+- Added collapsing sidebar for small screens (Douglas Muraoka, Manuel Trezza) [#1760](https://github.com/parse-community/parse-dashboard/pull/1760)
+- Added exclude hidden columns from being fetched from server (itzharDev) [#1694](https://github.com/parse-community/parse-dashboard/pull/1694)
+- Added exclude hidden columns from being fetched from server (Prerna Mehra) [#1734](https://github.com/parse-community/parse-dashboard/pull/1734)
```
Comment (sadakchap) [inline CHANGELOG.md:17]: I believe that, this pr was intended to solve a bug due to #1694 , which re-fetches the data only when a column is un-hidden  & its values are not cached.

File: src/dashboard/Data/Browser/BrowserTable.react.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ import PropTypes              from 'lib/PropTypes';
 
 const MAX_ROWS = 200; // Number of rows to render at any time
 const ROWS_OFFSET = 160;
-const ROW_HEIGHT = 31;
+const ROW_HEIGHT = 30;
```
Comment (davimacedo) [inline src/dashboard/Data/Browser/BrowserTable.react.js:24]: Is it intetional?

File: CHANGELOG.md (hunk 1 above)
Comment (mtrezza) [reply CHANGELOG.md]: Well spotted, I've moved it down to fixes and changed the description.

File: src/dashboard/Data/Browser/BrowserTable.react.js (hunk 2 above)
Comment (mtrezza) [reply src/dashboard/Data/Browser/BrowserTable.react.js:24]: I rebased on master, should be gone now.

**Review by davimacedo — APPROVED** [review]
LGTM!

**Review by sadakchap — APPROVED** [review]

**Comment by parseplatformorg** [comment]
🎉 This pull request has been released in version [3.1.0](https://github.com/parse-community/parse-dashboard/releases/tag/3.1.0)

**Pull request merged**